pystiche/papers/ulyanov_et_al_2016/loss.py

```diff
@@ -0,0 +1,182 @@
+from typing import Union, Optional, Sequence, Any, Dict
+from collections import OrderedDict
+import torch
+from pystiche.enc import Encoder, MultiLayerEncoder
+from pystiche.ops import (
+    MSEEncodingOperator,
+    GramOperator,
+    MultiLayerEncodingOperator,
+    TotalVariationOperator,
+)
+from pystiche.loss import MultiOperatorLoss
+from .utils import ulyanov_et_al_2016_multi_layer_encoder
+
+
```

Comment:
The important functions, i.e. the functions imported by `core.py` should be included in `__all__`.